components/Alert/Alert.unit.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Alert from './Alert';
+
+describe('Alert component', () => {
```

Comment:
I believe that this tests cover are suficient.
It you want to create a redundancy coverage, i would suggest a new two tests:
- verify if the children text passed is the same;
- verify when the callback onClose was set, if the icon close ir really is the close icon;

WIth Daniel in the input and textarea stories, we did it that way.
For a children case, for example:
```
    it('should the children text be the same at the prop', () => {
      const childrenContent = 'lorem ipsum ';
      const component = mount(
        <Alert onClose={() => {}}>{childrenContent}</Alert>,
      );
      const componentChildrenContent = component
        .find('Content')
        .childAt(0)
        .text();

      expect(componentChildrenContent).toMatch(childrenContent);
    });
```
I dont know if i was clear rsss

Reply:
Wow! Nice!